hackage-server.cabal

```diff
@@ -129,51 +130,52 @@ common defaults
   -- see `cabal.project.local-ghc-${VERSION}` files
   build-depends:
     , array                  >= 0.5   && < 0.6
-    , base                   >= 4.13  && < 4.20
+    , base                   >= 4.13  && < 4.21
     , binary                 >= 0.8   && < 0.9
     , bytestring             >= 0.10  && < 0.13
     , containers             >= 0.6.0 && < 0.8
     , deepseq                >= 1.4   && < 1.6
     , directory              >= 1.3   && < 1.4
-    , filepath               >= 1.4   && < 1.5
+    , filepath               >= 1.4   && < 1.6
     , mtl                    >= 2.2.1 && < 2.4
         -- we use Control.Monad.Except, introduced in mtl-2.2.1
     , pretty                 >= 1.1   && < 1.2
     , process                >= 1.6   && < 1.7
     , text                  ^>= 1.2.5.0 || >= 2.0 && < 2.2
-    , time                   >= 1.9   && < 1.13
+    , time                   >= 1.9   && < 1.15
```

Comment:
Was time-1.14 actually tested? time is different from other boot libraries because GHC-9.10 doesn't actually use the newest version. So e.g. assuming that that solving all of `cabal outdated` is necessary for GHC-9.10 is wrong. GHC-9.10 still uses time-1.12. I don't have an issue with bumping this, but then we need to make sure we're testing it.

Reply:
No wasn't tested, I can go back to 1.12.

Reply:
Restricted to `time < 1.13` again.